### DISCUSSION TITLE
Error when using helpers

### DISCUSSION THREAD

**Discussion by gungim** [StarArawn/bevy_ecs_tilemap#400] [Q&A] [answered]
I get this error when copying folder [helpers](https://github.com/StarArawn/bevy_ecs_tilemap/tree/main/examples/helpers) to my project
Error info:
```
rustc: the trait bound `bevy_ecs_tilemap::tiles::TileStorage: bevy::prelude::Component` is not satisfied
the following other types implement trait `bevy::prelude::Component`:
  Aabb
  AccessibilityNode
  Anchor
  BloomSettings
  Camera
  Camera2d
  Camera3d
  CameraFxaaPipeline
and 55 others
required for `&bevy_ecs_tilemap::tiles::TileStorage` to implement `ReadOnlyWorldQuery`
```
bevy version: 0.10
bevy_ecs_tilemap version: 0.9.0.
Someone please help me

**Comment by GeoffreyIanWard** [comment]
I am having a similar issue, and have not been able to find a solution. It's my understanding that bevy_ecs_tilemap 0.9.0 is compatible with bevy 0.10, so I'm not sure why this error is occurring. 
``` 
the trait bound `TilemapTexture: bevy::prelude::Component` is not satisfied
the following other types implement trait `bevy::prelude::Component`:
  Aabb
  AccessibilityNode
  AlphaMode
  Anchor
  AnimationPlayer
  BackgroundColor
  BloomSettings
  CalculatedClip
and 101 others
required for `&mut TilemapTexture` to implement `WorldQuery` 
```

**Reply by rparrett** [reply]
Please see https://github.com/StarArawn/bevy_ecs_tilemap/discussions/400#discussioncomment-5408925

**Comment by rparrett** [comment, marked as answer]
This type of error is indicative of there being multiple versions of Bevy in your dependecy tree.

In this case, the root cause is that `bevy_ecs_tilemap` has not released a version that is compatible with Bevy 0.10.

For now, it's possible to depend on the main git branch. Hopefully we'll have a crate published soon.
